instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -696,19 +709,23 @@ async def otel_send(message: dict[str, Any]):
             ) as send_span:
                 if callable(self.client_response_hook):
                     self.client_response_hook(send_span, scope, message)
+
+                status_code = None
+                if message["type"] == "http.response.start":
+                    status_code = message["status"]
+                elif message["type"] == "websocket.send":
+                    status_code = 200
+                if status_code:
```

Comment:
Before you were adding the status code only for this two message types and not for everything you get

Reply:
It's effectively the same, right? `status_code` defaults to `None` from :713 unless one of the two message types sets it.

Reply:
indeed, sorry for the noise :sweat: